doc/internals.rst

```diff
@@ -138,3 +138,49 @@ To help users keep things straight, please `let us know
 <https://github.com/pydata/xarray/issues>`_ if you plan to write a new accessor
 for an open source library. In the future, we will maintain a list of accessors
 and the libraries that implement them on this page.
+
+.. _zarr_encoding:
+
+Zarr Encoding Specification
+---------------------------
+
+In implementing support for the `Zarr <https://zarr.readthedocs.io/>`_ storage
+format, Xarray developers made some *ad hoc* choices about how to store
+NetCDF data in Zarr.
```

Comment:
I guess it's kind of implicit, but this might be the place to note something or add a link to some reference about ongoing efforts to standardize this.

Reply:
Addressed in latest commit.